lua/orgmode/objects/calendar.lua

```diff
@@ -46,13 +44,17 @@ function Calendar.new(data)
   this.on_day = data.on_day
   if data.date then
     this.date = data.date
-    this.month = this.date:set({ day = 1 })
   else
-    this.month = Date.today():start_of('month')
+    this.date = Date.today():start_of('month')
```

Comment:
Why default to start of month?

Reply:
Also part of the initial quickfix, now obsolete. Removed.